setup.Rmd

```diff
@@ -13,52 +13,36 @@ install.packages(c("devtools", "roxygen2", "testthat", "knitr"))
 ```
 
 Make sure you have a recent version of the RStudio integrated development environment (IDE).
-In fact, consider using the preview version and updating regularly.
-Compared to the official released version, the preview gives you access to the latest and greatest features and only slightly increases your chances of finding a bug.
-It is distinct from the more volatile daily build.
+New versions are released regularly, so we recommend updating often to get access to the latest and greatest features.
 
--   Preview version: <https://www.rstudio.com/products/rstudio/download/preview/>
--   Released version: <https://www.rstudio.com/products/rstudio/download/>
-    -   Most readers can use the free, open source version of RStudio Desktop.
+Download the current version of RStudio Desktop here: <https://posit.co/download/rstudio-desktop/>.
+Most readers can use the free, open source version of RStudio Desktop.
 
 ## devtools, usethis, and you {#sec-setup-usage}
 
 > "I am large, I contain multitudes."
 >
 > --- Walt Whitman, Song of Myself
 
-After 7 years of development, devtools had grown into a rather unwieldy package, making maintenance difficult.
-Version 2.0.0, released in late 2018, marked the [conscious uncoupling](https://www.tidyverse.org/articles/2018/10/devtools-2-0-0/) of devtools, with most functionality moving into seven smaller packages.
-Through various means, devtools continues to expose all its usual functionality, although it is mostly maintained elsewhere.
+As mentioned in @sec-intro-phil, devtools is a 'meta-package', encompassing and exposing functionality maintained in seven smaller packages[^setup-1].
 For example, devtools might provide a wrapper function in order to set user-friendly defaults, introduce helpful interactive behaviour, or to combine functionality from multiple sub-packages.
+In some cases it simply re-exports a function from another package to make it easily available when devtools is loaded.
```

Comment:
```suggestion
In some cases it simply re-exports a function from another package to make it easily available when devtools is attached.
```

Sort of a nitpick. We do play loose with load vs. attach in some places in the book, but mostly try to be pretty pedantic about it.

Reply:
Yes, great catch